dev_guide/templates/templates_load.adoc

```diff
@@ -0,0 +1,236 @@
+[[dev-guide-templates-loading]]
+= Loading Default Templates
```

Comment:
Much of this section is imported from install. The request was for  a "comprehensive" template guide, so it seemed to make sense (to me anyway) to include it. Any reasons not to that I am not aware of?

Reply:
because these aren't actions end users/developers on the system can or would take.

this is something the administrator needs to do.

Nothing from "install" belongs in the developer guide, it should stay in the administrative/install guide.

Reply:
@bparees What if we moved the Template topic out of the Developer Guide and create its own book? That way, we could include Admin and Dev tasks? See:
http://file.rdu.redhat.com/mburke/BZ-1258291/templates/template_guide.html

Reply:
no, i don't think that's the right approach.

The admin tasks related to templates are really just install tasks, they're very basic.

most of the template-related activities are developer-centric.